src/components/Tab/Tab.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './tab.scss';
+
+export const Tab = (props) => {
```

Comment:
Use destructuring for the props :
`export const Tab = ({ num
, onClick, index  }) => {
`